circle.yml

```diff
@@ -3,17 +3,30 @@ general:
     ignore:
       - gh-pages # list of branches to ignore
 machine:
+  environment:
+    YARN_VERSION: 0.20.3
+    PATH: "${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
   node:
-    version: 6.2.0
+    version: 4
```

Comment:
we don't really need to build the website with more than one version of Node. Can we do this only with 7?